packages/webapp/src/components/lists/ReportList/index.tsx

```diff
@@ -113,6 +140,12 @@ export const ReportList: StandardFC<ReportListProps> = wrap(function ReportList(
 		})
 	}
 
+	// using useEffect as "onComponentMount" to set hidden fields after initial fetch
+	useEffect(() => {
+		setHiddenFields(preferencesObj?.reportList[reportType]?.hiddenFields ?? {})
+		/* eslint-disable-next-line react-hooks/exhaustive-deps*/
```

Comment:
I wonder if we should remove this es-lint rule, because it has been tedious to have something run only once on mount? @karlolsonuc and @phorne-uncharted what do you think?